http/check_service.go

```diff
@@ -235,6 +237,28 @@ func (h *CheckHandler) handleGetChecks(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (h *CheckHandler) handleGetCheckQuery(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	h.Logger.Debug("check query retrieve request", zap.String("r", fmt.Sprint(r)))
```

Comment:
This could be pretty heavy weight.  If you want to debug something out of here, would you grab specific request fields?

Reply:
I copied from the previous model, shall we take care this issue in another ticket and update them all? I believe we will need the header, request body, the request url, not sure if we need the request ip, probably yes?

Reply:
I would remove it from this PR, yes.  The further on issue could talk about what kind of debug logs one would want in the API.